modules/dataframe/src/processors/csvsource.cpp

```diff
@@ -133,4 +156,92 @@ void CSVSource::deserialize(Deserializer& d) {
+                if (!cp->isChecked()) continue;
+
+                const auto& identifier = cp->getIdentifier();
+
+                if (startsWith(identifier, "emptyLine")) {
+                    rowFilters.push_back(csvfilters::emptyLines(
+                        detail::getValue<BoolProperty>(cp, "filterOnHeader")));
+                } else if (startsWith(identifier, "rowBegin")) {
+                    rowFilters.push_back(
+                        csvfilters::rowBegin(detail::getValue<StringProperty>(cp, "match"),
+                                             detail::getValue<BoolProperty>(cp, "filterOnHeader")));
+                } else if (startsWith(identifier, "lineRange")) {
+                    const auto& range = detail::getValue<IntMinMaxProperty>(cp, "range");
+                    rowFilters.push_back(csvfilters::lineRange(
+                        range.x, range.y, detail::getValue<BoolProperty>(cp, "filterOnHeader")));
+                } else if (startsWith(identifier, "stringItem")) {
+                    itemFilters.push_back(csvfilters::stringMatch(
+                        detail::getValue<IntProperty>(cp, "column"),
+                        detail::getValue<TemplateOptionProperty<filters::StringComp>>(cp, "comp"),
+                        detail::getValue<StringProperty>(cp, "match")));
+                } else if (startsWith(identifier, "intItem")) {
+                    itemFilters.push_back(csvfilters::intMatch(
+                        detail::getValue<IntProperty>(cp, "column"),
+                        detail::getValue<TemplateOptionProperty<filters::NumberComp>>(cp, "comp"),
+                        detail::getValue<IntProperty>(cp, "value")));
+                } else if (startsWith(identifier, "floatItem")) {
+                    itemFilters.push_back(csvfilters::floatMatch(
+                        detail::getValue<IntProperty>(cp, "column"),
+                        detail::getValue<TemplateOptionProperty<filters::NumberComp>>(cp, "comp"),
+                        detail::getValue<FloatProperty>(cp, "value"),
+                        detail::getValue<FloatProperty>(cp, "epsilon")));
+                } else if (startsWith(identifier, "doubleItem")) {
+                    itemFilters.push_back(csvfilters::doubleMatch(
+                        detail::getValue<IntProperty>(cp, "column"),
+                        detail::getValue<TemplateOptionProperty<filters::NumberComp>>(cp, "comp"),
+                        detail::getValue<DoubleProperty>(cp, "value"),
+                        detail::getValue<DoubleProperty>(cp, "epsilon")));
+                } else if (startsWith(identifier, "intRangeItem")) {
+                    const auto& range = detail::getValue<IntMinMaxProperty>(cp, "range");
+                    itemFilters.push_back(csvfilters::intRange(
+                        detail::getValue<IntProperty>(cp, "column"), range.x, range.y));
+                } else if (startsWith(identifier, "floatRangeItem")) {
```

Comment:
Would also be possible to use float/double comparison depending on if the doubleprecision checkbox is enabled. Then we would get rid of the floatXXX filters